test/framework/proxy.go

```diff
@@ -0,0 +1,37 @@
+package framework
+
+import (
+	"os"
+	"strings"
+
+	"github.com/aws/eks-anywhere/internal/pkg/api"
+)
+
+const (
+	httpProxyVar  = "T_HTTP_PROXY"
```

Comment:
Can we add these requirements to the README?

Reply:
done